ocaml/xapi/xapi_globs.ml

```diff
@@ -1075,6 +1077,7 @@ module Resources = struct
     "non-managed-pifs", non_managed_pifs, "Executed during PIF.scan to find out which NICs should not be managed by xapi";
     "update-issue", update_issue_script, "Running update-service when configuring the management interface";
     "killall", kill_process_script, "Executed to kill process";
+    "nbd-firewall-config", nbd_firewall_config_script, "Executed after NBD-related networking changes to configure the firewall for NBD";
```

Comment:
This name makes it sound like a file of configuration data, but it's good to have it match the script name.
I (we) should have thought of this when I named the script in the first place...

Reply:
So should I leave it as it is for consistency, because this is the name of the script?

Reply:
I was thinking the same... But if the script is called this way, then fine.

Reply:
Maybe worth changing afterwards, in a coordinated pair of pull-requests (here and in xenserver-specs).
New name might be "nbd-firewall-setter" or "nbd-firewall-tool" or "nbd-firewall-interfaces"?

Reply:
Sounds good. We were discussing with Rob that in the future we may want to have one, unified way of changing the firewall config.